widgets/dialogs.pas

```diff
@@ -278,6 +287,33 @@ procedure TMessageDialog.KeyDown(var Key: NativeInt; Shift: TShiftState);
   end;
 end;
 
+procedure TMessageDialog.Show;
+var
+  ownForm: TWForm;
+  curHeight: Integer;
+begin
+  inherited Show;
+  if not (Owner is TWForm) then
+    exit;
+  ownForm := TWForm(Owner);
+  Self.BeginUpdate;
+  try
+    curHeight := FMessageText.HandleElement.scrollHeight + CMinButtonHeight + CControlsSpacing * 8 + CTitileHeight;
+    if curHeight - CMinImageWidth > ownForm.Height then
+      curHeight := ownForm.Height - CMinImageWidth;
+    if curHeight < CMinDialogHeight then
+      curHeight := CMinDialogHeight;
+    Self.Height := curHeight;
+    fMessagePanel.Height := FMessageText.HandleElement.scrollHeight;
+    if Owner is TWForm then begin
+      ownForm := TWForm(Owner);
+      Self.Top := round(OwnForm.Height/2 - Self.Height/2);
```

Comment:
Style corrections:
- `Round` in PascalCase
- spaces between operators (thus `Self.Height / 2`)
- I know that Pascal is case insensitive, but please use the casing you used for the variable, here `ownForm`

Reply:
Fixed.